### PR TITLE
Incorporated Review Comments

### PR DESCRIPTION
**Pull request by shivangi8293**
- Change test methods name to make test more readable
 - Simplified code
 - Created setUp method in the test class to remove the redundant code
 - Added more testing scenarios in each test case

### CODE REVIEW

**Comment by vishalsinha21** [comment]
good job 👍

File: src/main/java/com/fizzbuzz/problem/FizzBuzzProblem.java [hunk 1]
```diff
@@ -2,27 +2,20 @@
```
Comment (dhruv-mahajan) [inline src/main/java/com/fizzbuzz/problem/FizzBuzzProblem.java:2]: Please remove .idea folder from master

Comment (shivangi8293) [reply src/main/java/com/fizzbuzz/problem/FizzBuzzProblem.java:2]: Removed the .idea folder.

**Comment by dhruv-mahajan** [comment]
All good.

**Pull request merged**